converter/output/rita/freqconn/main_test.go

```diff
@@ -0,0 +1,106 @@
+package freqconn_test
```

Comment:
These main_test files set up the integration tests. 

To run the integration tests use `sudo $(which go) test -v ./...`. Its unfortunate that docker requires root privileges to spin up a container, but the ability to write tests against a Mongo database was very valuable while developing this new feature. Note you do not need to use sudo if your user is in the docker group.